### PR TITLE
DRILL-6415: Fixed TestGracefulShutdown.TestRestApi test from timing out

### PR DESCRIPTION
**Pull request by dvjyothsna**
@sohami Please review

### CODE REVIEW

File: exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java [hunk 1]
```diff
@@ -204,19 +204,16 @@ public void testRestApi() throws Exception {
     try ( ClusterFixture cluster = builder.build();
           final ClientFixture client = cluster.clientFixture()) {
       Drillbit drillbit = cluster.drillbit("db1");
-      int port = drillbit.getContext().getConfig().getInt("drill.exec.http.port");
+      int port = drillbit.getWebServerPort();
       int grace_period = drillbit.getContext().getConfig().getInt(ExecConstants.GRACE_PERIOD);
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java]: Please change to `gracePeriod` here and in the code below.

File: exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java [hunk 2]
```diff
@@ -204,19 +204,16 @@ public void testRestApi() throws Exception {
...
-      while( port < 8049) {
-        URL url = new URL("http://localhost:"+port+"/gracefulShutdown");
-        HttpURLConnection conn = (HttpURLConnection) url.openConnection();
-        conn.setRequestMethod("POST");
-        if (conn.getResponseCode() != 200) {
-          throw new RuntimeException("Failed : HTTP error code : "
-                  + conn.getResponseCode());
-        }
-        port++;
+      URL url = new URL("http://localhost:"+port+"/gracefulShutdown");
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java]: Please add spaces between `+`-s here and in the code below.

File: exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java [hunk 3]
```diff
@@ -237,7 +234,7 @@ public void testRestApi() throws Exception {
       }
 
       Assert.assertTrue(listener.isDone());
-      Assert.assertEquals(1,drillbitEndpoints.size());
+      Assert.assertEquals(2,drillbitEndpoints.size());
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java]: Please add space after comma.

File: exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java [hunk 4]
```diff
@@ -255,22 +252,17 @@ public void testRestApiShutdown() throws Exception {
     try ( ClusterFixture cluster = builder.build();
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java]: Please remove space after `(`.

**Comment by arina-ielchiieva** [comment]
@dvjyothsna could you please address code review comments? The faster we fix failing test, the better :)

**Comment by dvjyothsna** [comment]
@arina-ielchiieva Looks like this is not fixing the issue. I need to investigate further. Closing this pull request

**Pull request closed without merging**